frontend/src/component/Dashboard/ContentTypeBtn.styl

```
@@ -3,9 +3,9 @@
 .contentTypeBtn
   display flex
   justify-content center
-  margin 15px 30px 0 0
+  margin standardSpacing 30px 0 0
```

Would it be possible to compute the top margin like this:
```suggestion
  margin standardSpacing calc(2*standardSpacing) 0 0
```
?